Source/WebCore/rendering/RenderListBox.h

```diff
@@ -65,6 +65,8 @@ class RenderListBox final : public RenderBlockFlow, public ScrollableArea {
     bool scrolledToBottom() const final;
     bool scrolledToLeft() const final;
     bool scrolledToRight() const final;
+    
+    bool isVisibleToHitTesting() const final { return true; };
```

Comment:
This needs to return `visibleToHitTesting()`

Reply:
Fixed.